aspnetcore/mvc/views/razor.md

```diff
@@ -442,41 +442,125 @@ public class _Views_Something_cshtml : RazorPage<dynamic>
 
 Later in this article, the section [Inspect the Razor C# class generated for a view](#inspect-the-razor-c-class-generated-for-a-view) explains how to view this generated class.
 
-<a name="using"></a>
+### @attribute
 
-### @using
+The `@attribute` directive adds the given attribute to the class of the generated page or view. The following example adds the `[Authorize]` attribute:
 
-The `@using` directive adds the C# `using` directive to the generated view:
+```cshtml
+@attribute [Authorize]
+```
 
-[!code-cshtml[](razor/sample/Views/Home/Contact9.cshtml)]
+::: moniker range=">= aspnetcore-3.0"
 
-### @model
+### @attributes
```

Comment:
I think it's better if we treat directives and directive-attributes separately. They have totally different usage and implications.

Reply:
I was just going by the language "Razor directives" in descriptions. I'll factor that into a new section called *Directive attributes*. I'll get a commit in now for that.